server/scripts/cli.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+import click
+
+
+@click.group()
+@click.version_option()
+def cli():
+    pass
+
+
+import scripts.commands  # noqa: E402 F401
```

Comment:
This is kind of hacky to import all the click commands and rely on the import registering all the click commands. There are much better ways to separate out the commands and cli files. 

The top answer in this stackoverflow post is pretty good:
<https://stackoverflow.com/questions/34643620/how-can-i-split-my-click-commands-each-with-a-set-of-sub-commands-into-multipl>

But I'd recommend doing something similar to what girder is doing:
<https://github.com/girder/girder/tree/master/girder/cli>

This also might be the reason why the pyinstaller build doesn't work.